src/shared/components/views/DatatableWithToggle.tsx

```diff
@@ -47,6 +55,28 @@ const DatatableWithToggle: FC = ({ children }) => {
     }
   }, [expandTable, params.accession]);
 
+  const sizeCheck = useCallback(() => {
+    setShowButton(
+      tableRef.current?.shadowRoot?.firstElementChild?.scrollHeight !==
+        tableRef.current?.shadowRoot?.firstElementChild?.clientHeight
+    );
+  }, []);
```

Comment:
Feels a bit hacky to rely on implementation detail of the datatable in Nightingale, but that works, and the datatable will eventually move away from Nightingale